src/nunit.analyzers.tests/DiagnosticSuppressors/DereferencePossiblyNullReferenceSuppressorTests.cs

```diff
@@ -638,9 +612,9 @@ public void HasCountShouldNotAffectNullabilitySuppression(List<int>? maybeNull)
                 }
```

Comment:
Line 610 we should add `↓` in `Assert.That(↓maybeNull[1], Is.EqualTo(1));`.

Reply:
Done.